src/Payments.Mvc/Controllers/TeamsController.cs

```diff
@@ -124,6 +124,7 @@ public async Task<IActionResult> Details(int? id)
 
             var model = new TeamDetailsModel
             {
+                Id                 = team.Id.ToString(),

```

Comment:
@jpknoll In my pr, I changed the Id to an int.

Reply:
Yeah I know. I did this work yesterday. I'm just going to use your changes when I resolve the conflict.